pkgs/top-level/all-packages.nix

```diff
@@ -2062,7 +2062,7 @@ in
 
   horst = callPackage ../tools/networking/horst { };
 
-  host = callPackage ../tools/networking/host { };
+  host = bind.host;
```

Comment:
`host` was historically and is still a separate package in most Linux distributions. I chose to go with the Debian naming where bind9 also provides the dnsutils and host binary packages: https://packages.qa.debian.org/b/bind9.html. Just noticed, though, that dnsutils actually depends on host.

Not sure about this, we can do it either way in my opinion.